Gems/Atom/RPI/Code/Source/RPI.Private/PerformanceCVarManager.cpp

```diff
@@ -0,0 +1,122 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+
+#include <AzCore/Debug/PerformanceCollector.h>
+
+#include "PerformanceCVarManager.h"
+
+namespace AZ
+{
+    namespace RPI
+    {
+
+        AZ::Debug::PerformanceCollector::DataLogType GetDataLogTypeFromCVar(const AZ::CVarFixedString& newCaptureType)
+        {
+            if (newCaptureType[0] == 'a' || newCaptureType[0] == 'A')
+            {
+                return AZ::Debug::PerformanceCollector::DataLogType::LogAllSamples;
+            }
+            return AZ::Debug::PerformanceCollector::DataLogType::LogStatistics;
+        }
+
+        class PerformanceCvarManager
+        {
+        public:
+            static constexpr char LogName[] = "RPIPerformanceCvarManager";
+
+            static void OnDataLogTypeChanged(const AZ::CVarFixedString& newCaptureType)
+            {
+                AZ_TracePrintf(LogName, "%s cvar changed to %s", __FUNCTION__, newCaptureType.c_str());
```

Comment:
A newline needs to be added to the `AZ_TracePrintf`
```suggestion
                AZ_TracePrintf(LogName, "%s cvar changed to %s\n", __FUNCTION__, newCaptureType.c_str());
```